apps/balancer-tools/src/app/apr/(components)/KPI.tsx

```diff
@@ -1,6 +1,12 @@
-export function KPI({ title, content }: { title: string; content: string }) {
+export function KPI({
+  title,
+  content,
+}: {
+  title: string;
+  content: string | JSX.Element;
```

Comment:
I think `React.ReactNode` is the most accepting, then you wouldn't even need `string` here.

Reply:
Actually had forgotten about `React.ReactNode`